qt/scientific_interfaces/Indirect/IndirectFunctionBrowser/SingleFunctionTemplateBrowser.cpp

```diff
@@ -173,6 +174,14 @@ void SingleFunctionTemplateBrowser::setParameterValue(
   m_parameterManager->setError(m_parameterMap[parameterName], parameterError);
 }
 
+void SingleFunctionTemplateBrowser::setParameterValueQuietly(
```

Comment:
SingleFunctionTemplateBrowser has been updated here. Do we want to make the same updates to the ConvTemplateBrowser and IqtTemplateBrowser as well. They will presumably be having the same speed issues?

Reply:
It appears those tabs are already sending the signals quietly, I only noticed the slow down on the MSD and F(q) tabs.